metricbeat/module/vsphere/host/_meta/fields.yml

```diff
@@ -35,3 +35,8 @@
       description: >
         Free Memory in bytes
       format: bytes
+    - name: network_names
+      type: object
```

Comment:
Why did you go for `object`? I'm wondering if this should be `keyword` as this will hold an array?